source/includes/_orders.md

```diff
@@ -132,71 +152,207 @@ currency | subresource | currency used in the order
+| last_name <br> <span>string</span>    | Last name of the customer               |
+| email <br> <span>string</span>        | Email address of the customer           |
+
+### Garment
+
+```json
+# Standard Object - Used in a resource collection
+{
+    "id": 892902,
+    "title": "ID-892902",
+    "order_id": 407781,
+    "copied_garment_id": null,
+    "price": "66.0",
+    "option_cost": "0.0",
+    "garment_type": "CSHT",
+    "created_at": "2018-08-17T09:50:45.000Z",
+    "updated_at": "2018-08-29T03:52:13.000Z",
+    "order_status": ...,
+    "delay_status": ...,
+    "dealer_order": ...
+}
+```
 
+Standard Attributes
 
+| Attribute                                   | Description                                                                                                      |
+| ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| id <br> <span>integer</span>                | Unique identifier for the object                                                                                 |
+| title <br> <span>string</span>              | Our SKU field which includes the garment id number and information about where the factory will send the garment |
+| order_id <br> <span>integer</span>          | Each garment must be a part of a specific order                                                                  |
+| copied_garment_id <br> <span>integer</span> | Garment id if this was copied from a previous order                                                              |
+| price <br> <span>decimal</span>             | Wholesale cost of the garment                                                                                    |
+| option_cost <br> <span>decimal</span>       | Dollar cost of the premium options used in this garment                                                          |
+| garment_type <br> <span>string</span>       | Type of garment. [Click here](#garment-types) for more info                                                      |
+| created_at <br> <span>datetime</span>       | When the garment was first created (but not ordered)                                                             |
+| updated_at <br> <span>datetime</span>       | When the garment was last modified                                                                               |
+| order_status <br> <span>subresource</span>  | Current status of the order. [Click here](#order-statuses) for more info                                         |
+| delay_status <br> <span>subresource</span>  | If garment is delayed, this will return why. [Click here](#delay-statuses) for more info                         |
+| dealer_order <br> <span>subresource</span>  | *Description TBD*                                                                                                |
```

Comment:
The full order placed by a single customer.  Includes this garment and can include more garments.